source/install/desktop.rst

```diff
@@ -54,11 +51,11 @@ The following Group Policies are available:
 macOS 10.9+
 --------------------------------------------------
 
-1. Download `latest version of the Mattermost desktop app <https://releases.mattermost.com/desktop/4.2.3/mattermost-desktop-4.2.3-mac.zip>`__.
+1. Download `latest version of the Mattermost desktop app <https://releases.mattermost.com/desktop/4.3.0/mattermost-desktop-4.3.0-mac.dmg>`__.
```

Comment:
Changed this to instructions for downloading and installing using the .dmg as the Github release doesn't list the .zip file. Can add that back in instead if desired!